src/azure-cli/azure/cli/command_modules/servicefabric/_params.py

```diff
@@ -66,7 +69,7 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('certificate_output_folder', options_list=['--certificate-output-folder', '--cert-out-folder'], help='The folder of the new certificate file to be created.')
         c.argument('certificate_password', help='The password of the certificate file.')
         c.argument('certificate_subject_name', options_list=['--certificate-subject-name', '--cert-subject-name'], help='The subject name of the certificate to be created.')
-        c.argument('vault_resource_group_name', options_list=['--vault-resource-group'], help='Key vault resource group name, if not given it will be cluster resource group name')
+        c.argument('vault_resource_group_name', options_list=['--vault-rg'], help='Key vault resource group name, if not given it will be cluster resource group name')
```

Comment:
changing parameter name is also breaking change to customers. There are two options for you:

1. Provide two options with `options_list=['--vault-resource-group', '--vault-rg']`.
2. Deprecate previous one and redirect to the new one. See `Deprecate Argument Option` part in https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#deprecating-commands-and-arguments

Reply:
thanks, done. I did this because some automatic check was failing. I'll remove it completely on the next breaking change release